components/dashboard/src/projects/Projects.tsx

```diff
@@ -122,27 +122,46 @@ export default function () {
                                     </div>
                                 </div>
-                                <p>{toRemoteURL(p.cloneUrl)}</p>
+                                <Link to={`/`}>
+                                    <p className="hover:text-gray-600 dark:hover:text-gray-400 dark:text-gray-500">{toRemoteURL(p.cloneUrl)}</p>
+                                </Link>
+                            </div>
+                            <div className="h-10 px-6 py-1 text-gray-400 text-sm">
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/`}>
+                                        Branches
+                                    </Link>
+                                </span>
+                                <span className="mx-2 my-auto">·</span>
+                                <span className="hover:text-gray-600 dark:hover:text-gray-300">
+                                    <Link to={`/`}>
+                                        Prebuilds
+                                    </Link>
+                                </span>
                             </div>
-                            <div className="h-1/4 px-6 py-1"><p>__ Active Branches</p></div>
                         </div>
-                        <div className="h-1/6 px-6 border rounded-b-xl dark:border-gray-800 bg-gray-200 cursor-pointer" onClick={() => viewAllPrebuilds(p)}>
+                        <div className="h-10 px-4 border rounded-b-xl dark:border-gray-800 bg-gray-100 border-gray-100 dark:bg-gray-800" onClick={() => viewAllPrebuilds(p)}>
                             {lastPrebuilds.get(p.id)
-                                ? (<div className="flex flex-row space-x-3 h-full text-sm">
-                                    <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
-                                        &nbsp;
-                                    </div>
-                                    <div className="my-auto">{lastPrebuilds.get(p.id)!.branch}</div>
-                                    <div className="my-auto text-gray-400">{moment(lastPrebuilds.get(p.id)!.startedAt, "YYYYMMDD").fromNow()}</div>
-                                    <div className="my-auto text-gray-400 flex-grow text-right">View All ⟶</div>
+                                ? (<div className="flex flex-row h-full text-sm justify-between">
+                                    <Link to={`/`} className="flex my-auto group space-x-2">
+                                        <div className={"my-auto rounded-full w-3 h-3 text-sm align-middle " + (true ? "bg-green-500" : "bg-gray-400")}>
+                                            &nbsp;
+                                        </div>
+                                        <div className="my-auto font-semibold text-gray-500">{lastPrebuilds.get(p.id)!.branch}</div>
```

Comment:
**suggestion**: Friendly reminder to truncate the branch name to scale better and avoid any text overflow. 🎗️

Reply:
Is that a self-reminder? 😅

Reply:
Possibly a simple `truncate` class could do the trick but haven't tried this with the new changes in this PR! 😇

Reply:
Maybe like this? (Untested)

```suggestion
                                        <div className="my-auto font-semibold text-gray-500 truncate" title={lastPrebuilds.get(p.id)!.branch}>{lastPrebuilds.get(p.id)!.branch}</div>
```